src/esp/gfx/replay/Player.cpp

```diff
@@ -19,6 +19,14 @@ void Player::readKeyframesFromJsonDocument(const rapidjson::Document& d) {
   esp::io::readMember(d, "keyframes", keyframes_);
 }
 
+Keyframe Player::keyframeFromString(const std::string& keyframe) {
+  Keyframe res;
+  rapidjson::Document d;
+  d.Parse<0>(keyframe.c_str());
+  esp::io::readMember(d, "keyframe", res);
```

Comment:
Is d trivial to copy? Otherwise, might  want to be:
```suggestion
  esp::io::readMember(std:move(d), "keyframe", res);
```